aten/src/ATen/native/native_functions.yaml

```diff
@@ -4579,6 +4576,8 @@
 
 - func: _softmax_backward_data(Tensor grad_output, Tensor output, int dim, ScalarType input_dtype) -> Tensor
   structured_delegate: _softmax_backward_data.out
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: nested_softmax_backward
```

Comment:
You need to leave the CompositeImplicitAutograd key here

Reply:
thanks good catch

Reply:
I added the key and am getting undefined symbol errors. I need to check and see if structured delegates stamp out the _softmax_backward_data() compostiteImplicit kernels.. I would think so but something seems to not be working here

Reply:
Still can't get this to work if I add the CompositeImplicitAutograd. I think because it is a structured delegate the dispatch table is in fact populated from the kernels defined from here:
```
- func: _softmax_backward_data.out(Tensor grad_output, Tensor output, int dim, ScalarType input_dtype, *, Tensor(a!) grad_input) -> Tensor(a!)
  structured: True
  dispatch:
    CPU: softmax_backward_cpu_out
    CUDA: softmax_backward_cuda_out
    MPS: softmax_backward_mps_out
```
so there is no `_softmax_backward_data` compositeImpllict kernel? or at least I am not sure what the name of that composite kernel is if it does exist, because it does not match the name `_softmax_backward_data`